dask/bag/text.py

```diff
@@ -103,6 +93,12 @@ def read_text(urlpath, blocksize=None, compression='infer',
         return from_delayed(blocks)
 
 
+def file_to_blocks(f):
+    with f as f:
+        for line in f:
+            yield line
```

Comment:
Interestingly we would normally handle the fusion explicitly by constructing the compound task as in `(list, (file_to_blocks, ...))`.  It's not clear how we do this with `delayed`.